app/src/main/res/values/strings.xml

```diff
@@ -5,19 +5,28 @@
   <string name="info_screen_name">Info</string>
   <string name="required">Required</string>
   <string name="optional">Optional</string>
+  <string name="read_more">Read more</string>
+  <string name="here">here</string>
+  <string name="result">Result</string>
+  <string name="none">None</string>
 
   <!-- Section titles -->
   <string name="section_title_sdk_initialization">SDK Initialization</string>
 
   <!-- SDK Initialization -->
+  <string name="sdk_initialization_description">First method that should be called on the Android SDK is the OneginiClient#start method. The method is responsible for asynchronous initialization of the OMI SDK.</string>
```

Comment:
I would not go into so much details in the description. I think text should be more vague to describe what is initialization, why is it needed and what is a result of this action.